lib/preface_make/freer_monad.ml

```diff
@@ -55,5 +55,20 @@ module Over (Type : Preface_specs.Types.T1) = struct
     ;;
   end)
 
+  module Selective =
+    Selective.Over_applicative_via_select
+      (Applicative)
+      (struct
+        type nonrec 'a t = 'a t
+
+        let select either f =
+          let open Monad in
+          either
+          >>= Either.fold
+                ~left:Applicative.((fun a -> ( |> ) a <$> f))
+                ~right:Applicative.pure
+        ;;
+      end)
```

Comment:
I was wondering why we didn't have a `Selective.From_monad` but that's because a Selective is also an Applicative. So it doesn't shock me to build it, however, there is `Selective.Select_from_monad` so I suggest using it! (And same for Free_monad).

Reply:
You are right. It is fixed